src/components/NcAppNavigationItem/NcAppNavigationItem.vue

```diff
@@ -765,7 +765,7 @@ export default {
 	/* counter */
 	.app-navigation-entry__counter-wrapper {
 		// Add slightly more space to the right of the counter
-		margin-right: 2px;
+		margin-right: 2px; /* same as NcAppNavigationCaption */
```

Comment:
I would to say to use a css variable here, although it is not possible to do it in a good way since the css of components are fully isolate.

Reply:
It uses a var now thanks to a conflict :)